arches_lingo/src/arches_lingo/components/scheme/report/SchemeNamespace.vue

```diff
@@ -110,6 +114,11 @@ function onNamespaceNameUpdate(val: string) {
         <div v-if="!mode || mode === VIEW">
             <SchemeReportSection
                 :title-text="$gettext('Scheme Namespace')"
+                :button-text="
```

Comment:
not sure about this. Since it's cardinality 1 should the button be hidden instead? Since I'm assuming the table row with have an edit interaction like cardinality-n tables